db/migrate/20200303143426_backfill_removals_removed_at.rb

```diff
@@ -0,0 +1,13 @@
+class BackfillRemovalsRemovedAt < ActiveRecord::Migration[6.0]
+  disable_ddl_transaction!
+
+  class Removal < ApplicationRecord
+    has_one :status, as: :details
+  end
+
+  def up
+    Removal.all.each { |removal| removal.update!(removed_at: removal.created_at) }
+  end
+
+  def down; end
```

Comment:
Minor, you don't need to define this.

Reply:
Yeah I didn't bother with `find_each` since there are only 6 records.

Reply:
Removed